package/MDAnalysis/core/universe.py

```diff
@@ -236,39 +236,56 @@ def universe(self):
-                 the "chained" reader has to guess the file format for each
-                 individual list member [``None``]
-                 Can also pass a subclass of :class:`MDAnalysis.coordinates.base.Reader`
-                 to define a custom reader to be used on the trajectory file.
-             *kwargs*
-                 Other kwargs are passed to the trajectory reader (only for advanced use)
-
-        :Returns: (filename, trajectory_format) or ``None`` if *filename* == ``None``
-        :Raises: :exc:`TypeError` if trajectory format can not be
+    def load_new(self, filename, format=None, in_memory=False, **kwargs):
+        """Load coordinates from *filename*, using the suffix to detect file
+        format.
+
+        Parameters
+        ----------
+        filename : str
+            the coordinate file (single frame or trajectory) *or* a list of
+            filenames, which are read one after another.
+        format : str (optional)
+            provide the file format of the coordinate or trajectory file;
+            ``None`` guesses it from the file extension. Note that this
+            keyword has no effect if a list of file names is supplied because
+            the "chained" reader has to guess the file format for each
+            individual list member [``None``] Can also pass a subclass of
+            :class:`MDAnalysis.coordinates.base.Reader` to define a custom
+            reader to be used on the trajectory file.
+        in_memory : bool (optional)
+            Directly load trajectory into memory
+        kwargs : dict
+            Other kwargs are passed to the trajectory reader (only for
+            advanced use)
+
+
+        # TODO: check what happens if filename is None
+        Returns
+        -------
+        filename : str
+        trajectory_format : str
+
+        # TODO: look up raises doc formating
+        Raises
+        ------
+        TypeError if trajectory format can not be
                   determined or no appropriate trajectory reader found
 
         .. versionchanged:: 0.8
-           If a list or sequence that is provided for *filename*  only contains a single entry
-           then it is treated as single coordinate file. This has the consequence that it is
-           not read by the :class:`~MDAnalysis.coordinates.base.ChainReader` but directly by
-           its specialized file format reader, which typically has more features than the
+           If a list or sequence that is provided for *filename* only contains
+           a single entry then it is treated as single coordinate file. This
+           has the consequence that it is not read by the
+           :class:`~MDAnalysis.coordinates.base.ChainReader` but directly by
+           its specialized file format reader, which typically has more
+           features than the
            :class:`~MDAnalysis.coordinates.base.ChainReader`.
+
         """
+        # TODO: is this really sensible? Why not require a filename arg?
```

Comment:
I think this is because it is blindly called from `Universe.__init__`.  It would make more sense to put the `None` check there and only possible call this.